src/Tokens/TokenRepository.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Statamic\Eloquent\Tokens;
+
+use Statamic\Contracts\Tokens\Token as TokenContract;
+use Statamic\Tokens\TokenRepository as FileTokenRepository;
+
+class TokenRepository extends FileTokenRepository
```

Comment:
You should be able to continue extending `Statamic\Tokens\TokenRepository`, but it has no "file" logic in it so maybe just change `as FileTokenRepository` to `as Repository`.